konflux-ci/namespace-lister/certificate.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: namespace-lister
+  namespace: namespace-lister
+spec:
+  isCA: true
+  subject:
+    organizations:
+      - konflux
```

Comment:
I don't think this field is necessary, especially since we shouldn't be issuing certificates.